packages/sparkling-controls/src/EditBox.svelte

```diff
@@ -12,33 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 -->
-
 <script lang="ts">
-  export let width: string = '300px'
-  export let label: string = ''
-  export let value: string
-  export let placeholder: string
-  export let id: string
-  export let hoverState: boolean = false
+  export let width: string = "300px";
```

Comment:
We prefer to not use ; if possible.

Reply:
Yeah, overlooked that during experimenting with prettier :)